sdk-config.json

```diff
@@ -20,57 +19,26 @@
     "infinityRestServerUrl_comment": "Full path to Infinity REST server",
     "infinityRestServerUrl": "https://localhost:1080/prweb",
 
-    "appAlias_comment": "appAlias of the application which operators will be accessing (e.g., MediaCo)",
```

Comment:
Do not modify this set of lines.  These settings are "more important" and shouldn't be adjusted.  Was only looking to remove the stuff above where we seem to continuously have to add an attribute to the ever growing list of assorted possible attributes in each of the SDK config files.  Wasn't expecting any other adjustments within the rest of the file.

Reply:
Done.